src/lib/dialog/Dialog.svelte

```
@@ -0,0 +1,147 @@
+    if (typeof onaction === "function" && onaction({ action: dlg.returnValue, data: new FormData(ev.target as HTMLFormElement) }) === false) return;
+
+    close(dlg);
+  }
+
+  function _ontoggle(ev: ToggleEvent & { currentTarget: HTMLDialogElement }) {
+    ontoggle?.(ev);
+    open = ev.newState === "open"; // for cases when toggle by other means
+  }
+
+  function init(dlg: HTMLDialogElement) {
+    modal ? dlg.showModal() : dlg.show();
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)